src/material-experimental/mdc-dialog/dialog-container.ts

```diff
@@ -0,0 +1,183 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {FocusTrapFactory} from '@angular/cdk/a11y';
+import {DOCUMENT} from '@angular/common';
+import {
+  AfterViewInit,
+  ChangeDetectionStrategy,
+  ChangeDetectorRef,
+  Component,
+  ElementRef,
+  Inject,
+  NgZone,
+  OnDestroy,
+  Optional,
+  ViewEncapsulation
+} from '@angular/core';
+import {MatDialogConfig, _MatDialogContainerBase} from '@angular/material/dialog';
+import {ANIMATION_MODULE_TYPE} from '@angular/platform-browser/animations';
+import {cssClasses, numbers} from '@material/dialog';
+
+/**
+ * Internal component that wraps user-provided dialog content in a MDC dialog.
+ * @docs-private
+ */
+@Component({
+  selector: 'mat-mdc-dialog-container',
+  templateUrl: 'dialog-container.html',
+  styleUrls: ['dialog.css'],
+  encapsulation: ViewEncapsulation.None,
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  host: {
+    'class': 'mat-mdc-dialog-container mdc-dialog',
+    'tabindex': '-1',
+    'aria-modal': 'true',
+    '[attr.id]': '_id',
```

Comment:
Just `[id]`?

Reply:
This is from the non-MDC dialog. Switched both to `[id]` now. Can't think of a reason why it has been done that way.